test/xspec.bats

```diff
@@ -193,6 +193,10 @@ teardown() {
 
 
 @test "executing the Saxon XProc harness generates a report with UTF-8 encoding" {
+	if [ -z ${XMLCALABASH_VERSION} ]; then
```

Comment:
The check in ```xspec.bats``` is now based on XMLCALABASH_CP instead of XMLCALABASH_VERSION (which makes more sense because XMLCalabash is properly installed only once XMLCALABASH_CP is set). 

Have you got any further changes?